admin/installer/_images.py

```diff
@@ -3,7 +3,60 @@
 """
 Helper utilities for CloudFormation Installer's Packer images.
 """
-from pyrsistent import freeze
+import json
+import sys
+from tempfile import mkdtemp
+
+import boto3
+
+from effect import (
+    Effect, ComposedDispatcher, TypeDispatcher,
+    sync_performer, base_dispatcher,
+)
+from effect.do import do, do_return
+
+from txeffect import deferred_performer, perform as async_perform
+
+from pyrsistent import PClass, field, freeze, thaw, pvector_field
+
+from twisted.python.filepath import FilePath
+from twisted.python.usage import Options, UsageError
+
+from flocker.common.runner import run
+
+
+PACKER_TEMPLATE_DIR = FilePath(__file__).sibling('packer')
+
+# The AWS regions supported by Packer.
+# XXX ap-northeast-2 is not supported by the current packer release:
+# https://github.com/mitchellh/packer/issues/3058
+AWS_REGIONS = (
```

Comment:
:grey_question: It'd be nice if there was a packer command you could run to get all of the regions that it thinks are valid. Looks like it's just a private array in a method though: https://github.com/chulkilee/packer/commit/1aa24035fcd287f7ad00f0366f4617b9230b3ddd#diff-743b70f36793a4f1edd68908b6048ef1L6

So I guess this is really just #moaning . Thanks for leaving a note to the github issue.